### PR TITLE
feat(Desktop)!: Updating desktop app due to IconSettings refactor

### PR DESCRIPTION
**Pull request by alexandraB99**
As part of https://github.com/status-im/StatusQ/issues/781
Depends on: https://github.com/status-im/StatusQ/pull/788

### What does the PR do
Updating desktop app due to IconSettings refactor

### Affected areas
Desktop app

### Screenshot of functionality (including design for comparison)
https://user-images.githubusercontent.com/31625338/184139460-7a7c9f45-d7f0-4e11-98c6-a167d5a4de83.mov

### CODE REVIEW

**Comment by caybro** [comment]
@alexandraB99 needs a rebase

**Review by caybro — APPROVED** [review]
Looks good, nice work!

File: ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml [hunk 1]
```diff
@@ -55,20 +55,13 @@ StackLayout {
...
-                        height: 80
-                        source: root.logoImageData
-                    }
+                    asset.width: 80
+                    asset.height: 80
+                    asset.isLetterIdenticon: !root.logoImageData
+                    asset.color: root.color
+                    asset.letterSize: width / 2.4
+                    asset.source: root.logoImageData
+                    asset.isImage: !!root.logoImageData
```
Comment (Khushboo-dev-cpp) [inline ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml:64]: remove !!? not very efficient?
root.logoImageData  !== ""  is better?

File: ui/imports/shared/views/SendModalHeader.qml [hunk 2]
```diff
@@ -85,12 +85,12 @@ StatusFloatingButtonsSelector {
...
-        icon.color: model.color
-        icon.name: !emoji ? "filled-account": ""
-        icon.letterSize: 14
-        icon.isLetterIdenticon: !!model.emoji
-        icon.background.color: Theme.palette.indirectColor1
+        asset.emoji: !!emoji ? emoji: ""
+        asset.color: model.color
+        asset.name: !emoji ? "filled-account": ""
+        asset.letterSize: 14
+        asset.isLetterIdenticon: !!model.emoji
```
Comment (Khushboo-dev-cpp) [inline ui/imports/shared/views/SendModalHeader.qml:92]: can isImage and isLetterIdenticon be made into a single Enum? making it mutually exclusive? this maybe confusing to use?

**Review by Khushboo-dev-cpp — APPROVED** [review]
LGTM just 2 small things. The Enum part is just something to think about

File: ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml (hunk 1 above)
Comment (alexandraB99) [reply ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml]: Removed, however I think !! works in both undefined and empty cases, isn't it?

File: ui/imports/shared/views/SendModalHeader.qml (hunk 2 above)
Comment (alexandraB99) [reply ui/imports/shared/views/SendModalHeader.qml:92]: might be. but also might be that none is needed, if we internally check whether the requested icon name exists in icons, if not then it's an image.. however this requires more work, would it be ok to create a new issue?

Comment (Khushboo-dev-cpp) [reply ui/imports/shared/views/SendModalHeader.qml:92]: sure :)

File: ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml (hunk 1 above)
Comment (Khushboo-dev-cpp) [reply ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml]: I am not sure :?

**Comment by glitchminer** [comment]
## Issues found

### 1. OPEN Icons missing from "More" menu
Actual
image

---
### 2. OPEN Icons missing from menu on clicking user
Actual - Contact list
image

Actual - Community and 1-1 Chat
image

---
### 3. OPEN Profile picture not shown in community participants list
Actual
image

image

File: ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml (hunk 1 above)
Comment (PascalPrecht) [reply ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml]: What @alexandraB99 suggested is correct. While `!== ""` checks for non empty string `!!expr` checks for truthyness
A non-empty string is falsy in JS. So is `undefined` and `null`

**Comment by alexandraB99** [comment]
@glitchminer thanks for testing! Issues 1&2 are now fixed. Regarding issue 3 it's about privacy settings that prevent showing the image, if I look at the code:
```
if ((!model.isContact &&
                    Global.privacyModuleInst.profilePicturesVisibility !==
                    Constants.profilePicturesVisibility.everyone)) {
                    return "";
}
```
if I comment out this snippet I can see the profile image:
m

**Comment by alexandraB99** [comment]
> @alexandraB99, thanks for the fixes. Closed items 1,2,3. Seems like some regression happened so had to add 4, and 5 here: [#6979 (comment)](https://github.com/status-im/status-desktop/pull/6979#issuecomment-1233195049)

@glitchminer 4&5 fixed! thanks

**Review by glitchminer — APPROVED** [review]
Tested and approved

**Pull request merged**